ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Use this template for reporting bugs.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
+
+### Bug Report
+
+#### Describe how to reproduce the bug
+
+Please provide a detailled description of what you have to do to reproduce your bug (settings, click paths, ...)
+
+#### Describe your SCM-Manager setup
+
+What plugins and what versions do you use? Are you using a server like tomcat? Do you have special settings like proxies? Are you running your instance behind a reverse proxy?
+
+#### Which operating system, browser and versions do you use?
+
+Windows, Mac or Linux? Chrome, Firefox or IE? What versions?
+
+#### Log of the bug
+
+Please mind to check for any sensible information like repository names!
```

Comment:
`[...] in any logs you provide.`
To make it clear where to check for the sensible information.